docs/api-bridging/spec.md

```diff
@@ -0,0 +1,1193 @@
+            "app": {
+               "name": "someOtherApp",
+               "appId": "",
+                "version": "",
+                "desktopAgent": "agent-B"
+                // ... other metadata fields
+           }
+       }
+    }
+}
+```
+
+The bridge will add in the source agent (agent-A) and forward the message to destination (agent-B)
+
+```JSON
+// DAB -> agent-B
+{
+    "type": "privateChannelDisconnect",
+    "payload": {},
+    "meta": {
+        "requestGuid": "requestGuid",
+        "timestamp": "2020-03-...",
+        "source": {
+            "name": "AChatApp",
+            "appId": "",
+            "version": "",
+            "desktopAgent": "agent-A"
+            // ... other metadata fields
+        },
+        "destination": { // duplicates the app argument
+            "app": {
+               "name": "someOtherApp",
+               "appId": "",
+                "version": "",
+                "desktopAgent": "agent-B"
+                // ... other metadata fields
+           }
+       }
+    }
+}
+```
+
+### open
+
+```typescript
+open(app: TargetApp, context?: Context): Promise<AppMetadata>;
+```
+
+When receiving a response from invoking `fdc3.open` the new app instances MUST be fully initialized ie. the responding Desktop Agent will need to return an `AppMetadata` with an `instanceId`.
+
+#### Request format
+
+A `fdc3.open` call is made on agent-A.
+
+```javascript
+// Open an app without context, using the app name
+let instanceMetadata = await fdc3.open('myApp');
+
+// Open an app without context, using an AppMetadata object to specify the target
+let appMetadata = {name: 'myApp', appId: 'myApp-v1.0.1', version: '1.0.1'};
+let instanceMetadata = await fdc3.open(appMetadata);
+
+// Open an app without context, using an AppMetadata object to specify the target and Desktop Agent
+let appMetadata = {name: 'myApp', appId: 'myApp-v1.0.1', version: '1.0.1', desktopAgent:"DesktopAgentB"};
+let instanceMetadata = await fdc3.open(appMetadata);
+```
+
+The `fdc3.open` command should result in a single copy of the specified app being opened and its instance data returned, or an error if it could not be opened. There are two possible scenarios:
+
+1) The Desktop Agent that the app should open on is specified
+2) The Desktop Agent that the app should open on is NOT specified app
+
+The first case (target Desktop Agent is specified) is simple:
+
+* If the local Desktop Agent is the target, handle the call normally
+* If you are a server
+  * check if any of your clients is the target and transmit the call to them and await a response
+  * If you are also a client of another server follow the client steps
+  * otherwise return `OpenError.AppNotFound`
+* If you are a client
+  * transmit the call to the server and await a response
+
+The second case is a little trickier as we don't know which agent may have the app available:
+
+* If the local Desktop Agent has the app, open it and exit.
+* If you are a server

```

Comment:
This will need to be reworded if we are using the bridge instead of client/server approach